tensorboard/plugins/hparams/api_test.py

```diff
@@ -257,16 +270,18 @@ def setUp(self):
         tf.keras.layers.Dense(1, activation="sigmoid"),
     ])
     self.model.compile(loss="mse", optimizer=self.hparams["optimizer"])
-    self.logdir = os.path.join(self.get_temp_dir(), "logs")
-    self.callback = hp.KerasCallback(
-        self.logdir,
-        self.hparams,
-        group_name="psl27",
-    )
+    self.callback = hp.KerasCallback(writer, self.hparams, group_name="psl27")
 
-  @test_util.run_v2_only("Requires eager mode.")
   def test_eager(self):
-    self.model.fit(x=[(1,)], y=[(2,)], callbacks=[self.callback])
+    def mock_time():
+      mock_time.time += 1
+      return mock_time.time
+    mock_time.time = 1556227801.875
+    initial_time = mock_time()
```

Comment:
might as well use `mock_time.time` here for a tighter bound?  Ditto for the end time.

Reply:
Eh… sure. I have to be careful about equality strictness then. :-)

Reply:
Not sure I follow?  It should be literally the same floating point value, no?

Reply:
Right, just `<=`/`>=` vs. `<`/`>`. Not a problem.